commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AbstractAuditLogSyslogReliableProtocolCommand.java

```diff
@@ -0,0 +1,105 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+abstract class AbstractAuditLogSyslogReliableProtocolCommand extends AbstractAuditLogSyslogHandlerCommand {
```

Comment:
I'm not super-excited about this class, but if it makes sense, then keep it in place. I only found that there was something wrong :-) Once you fix it, we can re-evaluate.

Oh and BTW, there's `recconect` a couple of lines below -- that's wrong, should be `reconnect` :-)